### PR TITLE
Ignore "ambiguous argument '(unreadable git hash).."

### PR DESCRIPTION
**Pull request by kalikiana**
See: https://progress.opensuse.org/issues/113030

### CODE REVIEW

File: logwarn_openqa [hunk 1]
```diff
@@ -78,6 +78,8 @@ $logwarn ${options} -m '\[.*:?(debug|info|warn|error)\]' -p ${file} \
     '!\[warn\].* Unable to wakeup scheduler: Request timeout' \
     `# https://progress.opensuse.org/issues/105918` \
     '!\[warn\].* fatal: Invalid revision range .*\.\.' \
+    `# https://progress.opensuse.org/issues/113030` \
+    `!\[warn\].* fatal: ambiguous argument \'\(unreadable git hash\)\.\.\(unreadable git hash\)\': unknown revision or path not in the working tree.` \
```
Comment (okurz) [inline logwarn_openqa:82]: please generalize to cover `\(unreadable git hash\)\.\.` as we have cases where the later job has a hash but the former doesn't

Comment (kalikiana) [reply logwarn_openqa:82]: Seems like the is-ignored unit test is broken. I added the other case and it's passing w/o further changes 🤔 Need to investigate this more

Comment (kalikiana) [reply logwarn_openqa:82]: Even `is-ignored '\[error\].+fatal: ambiguous argument e737020f1b7d8ff3f6cc3bbc009f83a4a09862cb'` also passes without further changes

Comment (kalikiana) [reply logwarn_openqa:82]: Aha! Apparently the wrong quotation marks cause the problem

**Review by okurz — APPROVED** [review]

**Review by okurz — APPROVED** [review]

File: logwarn_openqa (hunk 1 above)
Comment (kalikiana) [reply logwarn_openqa]: Solved with @perlpunk's help

**Pull request merged**